Content.Shared/Silicons/StationAi/SharedStationAiSystem.cs

```diff
@@ -226,6 +241,44 @@ private void OnHolderInteract(Entity<StationAiHolderComponent> ent, ref AfterInt
         }
     }
 
+    private void OnHolderInteract(Entity<StationAiHolderComponent> ent, ref AfterInteractEvent args)
+    {
+        if(!args.CanReach)
+            return;
+        
+        if (!TryComp(args.Target, out StationAiHolderComponent? targetHolder))
+            return;
+        
+        if (!TryComp(args.Used, out IntellicardComponent? intelliComp))
+            return;
+        
+        bool isUploading = _slots.CanEject(ent.Owner, args.User, ent.Comp.Slot);
+        
+        bool isDownloading = _slots.CanEject(args.Target.Value, args.User, targetHolder.Slot);
```

Comment:
```suggestion
        var isUploading = _slots.CanEject(ent.Owner, args.User, ent.Comp.Slot);
        var isDownloading = _slots.CanEject(args.Target.Value, args.User, targetHolder.Slot);
```
Use var instead of explicit type whe apparent
this is the style convention we use (and your IDE will tell you about 😄)